src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -186,7 +188,27 @@ public StrykerRunResult Test(StrykerOptions options)
                 }
             }
 
-            var mutantsToTest = mutantsNotRun.Where(x => x.ResultStatus != MutantStatus.Ignored && x.ResultStatus != MutantStatus.NoCoverage);
+            var mutantsToTest = mutantsNotRun;
+            if (_options.Optimizations.HasFlag(OptimizationFlags.CoverageBasedTest))
+            {
+                var testCount = _mutationTestExecutor.TestRunner.DiscoverNumberOfTests();
+                var toTest = @mutantsNotRun.Sum(x => x.MustRunAgainstAllTests ? testCount : x.CoveringTests.Count);
+                var total = testCount * viableMutantsCount;
+                if (total > 0 && total != toTest)
+                {
+                    _logger.LogInformation($"Coverage analysis eliminated {(total-toTest)/(double)total:P1} useless tests.");
```

Comment:
```suggestion
                    _logger.LogInformation($"Coverage analysis eliminated {(total-toTest)/(double)total:P1} of tests because they do not cover a mutant.");
```

Reply:
Actually, this is not about eliminating tests that do not cover mutants. Here it is about keeping only the tests that covers the mutant you are testing, so not sure how to translate that

Reply:
How do you feel about my update?

Reply:
It is better, but I think it is still misguiding
What I attempted here is to convey an overall impression of how much time the coverage analysis  feature helped saved. I kept both messages similar for consistency. They are both true, but do not describe precisely what is happening.

In both cases, the algorithm is compute the percentage of tests that Stryker will not run (thanks to coverage analysis). The initial number of needed tests is:
number of tests in the assembly(ies) * number of mutations, quite a large number for a significant project (roughly 4 millions for NFluent as an example).
Bear in mind this figure is provided for the whole test phase, not per mutant.

With `perTest` coverage analysis, each mutation will only be tested with the test covering it. Other tests will not be run, except when we cannot reliably determine the needed tests, the so called _static mutations_. 

Here we skip all tests that provide no information on the active mutation. In practice, I simply count how many tests will be run.

With 'all' coverage analysis, we will run every tests against every test mutations, assuming the mutation is tested by at least one test. Since the 'all' mode mostly make sense for `dotnet test`, where the number of tests is not available, I use the number of test sessions to compute the percentage. And it stil provides an accurate result, since each test session will run all existing tests.

**Ultimately**: what message do you want to convey with this information?

Reply:
My main issue with the original message you proposed is that 'useless' has a negative value connotation. It could appear to the user that their unit tests are useless but that is not the case.

Furthermore it is not explaining why the tests are eliminated, so in that case the message is quite useless to them.

I would like the message to convey to the user 'hey you have this feature turned on, it makes the mutation testrun this much faster because we don't have to run 98% of tests that would not have impact the result, but would have been run anyway if this feature was not turned on'

Reply:
I like this phrasing, I will make a proposal

Reply:
check the latest comit